pkg/services/alerting/notifier_test.go

```diff
@@ -269,6 +299,19 @@ func newTestNotifier(model *models.AlertNotification) (Notifier, error) {
 	}, nil
 }
 
+func newTestRule(name, description string) *Rule {
+	return &Rule{
+		ID:            1,
+		DashboardID:   1,
+		PanelID:       1,
+		OrgID:         1,
+		Name:          name,
+		Message:       description,
```

Comment:
It's less confusing if the parameter is named `message`, same as the field it initializes:
```suggestion
func newTestRule(name, message string) *Rule {
	return &Rule{
		ID:            1,
		DashboardID:   1,
		PanelID:       1,
		OrgID:         1,
		Name:          name,
		Message:       message,
```